src/travel_tracker_service.js

```diff
@@ -19,6 +19,15 @@ async function isValidTraveler(usernameInput) {
   return valid;
 }
 
+async function getAllTravelers() {
+  const url =
```

Comment:
sometimes i will see stuff like

baseUrl = "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/"

then each api call will be 

fetch(baseUrl + travelersUrl)

then you can modularly call different urls throughout the file without them all being super long and repeating the base URL, more DRY